docs/guides/variables/aws/ssm.md

```diff
@@ -11,9 +11,90 @@ keywords: ['Serverless Framework', 'AWS SSM', 'Secrets Manager', 'Variables']
 
 <!-- DOCS-SITE-LINK:END -->
 
-# Reference AWS SSM Parameter Store & Secrets Manager
+# Configuration options
 
-You can reference SSM Parameters as the source of your variables with the `ssm:/path/to/param` syntax. For example:
+| Option         | Required |  Type  |              Default               | Description                                                                 |
+| -------------- | :------: | :----: | :--------------------------------: | :-------------------------------------------------------------------------- |
+| `region`       |    No    | String | Inherited from parent AWS resolver | AWS region                                                                  |
+| `rawOrDecrypt` |    No    | String |                                    | `raw` or `noDecrypt` instruction to disable auto-parsing or auto-decrypting |
+
+## Examples
+
+### Default
+
```

Comment:
I think it'd be helpful to provide a little more context in each of these examples to explain why you would do this and how it works.

For example, the "Custom region" example should explain that `awsAccount1` will be used for deployment since it is the only `type: aws` provider, and `euSsm` will reference SSM parameter in a different region than the deploying region.